test/i18n/i18n_test.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class I18nExtensionsTest < ActiveSupport::TestCase
+  test "it correctly translates using .translate and .translate_to_original" do
+    assert_equal "Italienisch", I18n.t("locales.italian", locale: :de)
+
+    Moirai::Translation.create!(locale: "de", key: "locales.italian", value: "Italianese")
+
+    assert_equal 1, Moirai::Translation.count
+    assert_equal "Italianese", I18n.t("locales.italian", locale: :de)
```

Comment:
this is failing on the CI... but the above passes 🙃  so the record is there. Any ideas? Passes locally